server/src/packages/packages.controller.ts

```diff
@@ -119,22 +121,40 @@ import { JsonApiSerializeInterceptor } from '../json-api-serialize.interceptor';
   // remap verbose navigation link names to
   // more concise names
   transform(projectPackage) {
-    return {
-      ...projectPackage,
-      'pas-form': {
-        ...projectPackage?.dcp_pasform,
-        applicants: projectPackage?.dcp_pasform?.dcp_dcp_applicantinformation_dcp_pasform,
-        bbls: projectPackage?.dcp_pasform?.dcp_dcp_projectbbl_dcp_pasform,
-      },
-    };
+    const { dcp_pasform: pasForm } = projectPackage;
+
+    if (!pasForm) {
+      return {
+        ...projectPackage,
+      };
+    } else {
+      return {
+        ...projectPackage,
+        'pas-form': {
+          ...pasForm,
+          applicants: pasForm.dcp_dcp_applicantinformation_dcp_pasform,
```

Comment:
Just flagging that we're going to need to (at some point) specify the two CRM tables that our `applicants` will be sent to / retrieved from.  

_Think_ the name of the other applicant array is: `dcp_dcp_applicantrepresentativeinformation_dcp_pasform` but gotta double check that.

Reply:
Ahhh yes yes you are correct.

Thank you.

I will make a follow up issue for this. We need to merge those things. Might be messy.